vulcanai2/datasets/tabulardataset.py

```diff
@@ -67,41 +108,51 @@ def list_all_features(self):
         """
         return list(self.df)
 
-    def get_all_numeric_features(self):
+    def list_all_numeric_features(self):
         """
         Returns all columns that contain numeric values
         :return: all columns that contain numeric values
         """
-        return [key for key in dict(self.df.dtypes) if dict(self.df.dtypes)[key] in ['float64', 'int64']]
+        return [key for key in dict(self.df.dtypes) if dict(self.df.dtypes)[key] in ['float64', 'int64', 'float32', 'int32']]
 
-    def get_all_categorical_features(self):
+    def list_all_categorical_features(self):
         """
         Returns all columns that contain categorical values
         :return: all columns that contain categorical values
         """
-        return [key for key in dict(self.df.dtypes) if dict(self.df.dtypes)[key] not in ['float64', 'int64']]
+        return [key for key in dict(self.df.dtypes) if dict(self.df.dtypes)[key] not in ['float64', 'int64', 'float32', 'int32']]
 
     #TODO: check this doesn't operate in place... damn
     def remove_majority_null(self, threshold):
```

Comment:
With remove_majority_null and remove_unique, let's also make a note to the user of which columns were removed. This won't need to be the default but we can use our logger to specify the level of detail required from these functions. We want to be super transparent about what we remove and why when automating it. Looks great though :)